src/jats/jats-exporter.ts

```diff
@@ -1749,57 +1752,29 @@ export class JATSExporter {
           }
         })
       }
-      const noteIDs: string[] = []
-      for (const contributor of [...authorContributors, ...otherContributors]) {
-        if (contributor.footnote) {
-          const ids = contributor.footnote.map((note) => {
-            return note.noteID
-          })
-          noteIDs.push(...ids)
-        }
-        if (contributor.corresp) {
-          const ids = contributor.corresp.map((corresp) => {
-            return corresp.correspID
-          })
-          noteIDs.push(...ids)
-        }
-      }
-      const footnotes: Footnote[] = []
-      footnotes.push(
-        ...this.models.filter(hasObjectType<Footnote>(ObjectTypes.Footnote))
+      const authorNotes = this.models.find(
+        hasObjectType<AuthorNotes>(ObjectTypes.AuthorNotes)
```

Comment:
You can use `getModelsByType` instead.

Reply:
The reason i used `find` here is to get a single model in the result, and not an array.